KsApi/queries/BuildPaymentPlanQuery.graphql

```diff
@@ -8,12 +8,7 @@ query BuildPaymentPlan($slug: String!, $amount: String!) {
           amount
           currency
         }
-        id
```

Comment:
We never used these fields, cleaned them up to make testing slightly easier on myself.